### PR TITLE
Unused widgets are removed.

### PR DESCRIPTION
**Pull request by tasomaniac**
I don't know why but these are unused.
We can always retrieve them back if we want to. What do you say? Should we remove them?

### CODE REVIEW

**Comment by HenriqueRocha** [comment]
LGTM :+1:

File: app/src/main/java/org/gdg/frisbee/android/view/GdgCalendarView.java [hunk 1]
```diff
@@ -1,88 +0,0 @@
-/*
```
Comment (stefanhoth) [inline app/src/main/java/org/gdg/frisbee/android/view/GdgCalendarView.java]: @friedger @mauimauer Any insights on why this Class is not used anymore?

File: app/src/main/java/org/gdg/frisbee/android/view/MasonryView.java [hunk 2]
```diff
@@ -1,2555 +0,0 @@
...
-        int viewType;
-
-        /**
-         * The column this view is occupying
-         */
-        int column;
-
-        /**
-         * The stable ID of the item this view displays
-         */
```
Comment (stefanhoth) [inline app/src/main/java/org/gdg/frisbee/android/view/MasonryView.java]: Line 2555

File: app/src/main/java/org/gdg/frisbee/android/view/MyView.java [hunk 3]
```diff
@@ -41,8 +41,6 @@
  */
 public class MyView extends AbsListView {
 
-    private final static String LOG_TAG = "GDG-MyView";
-
```
Comment (stefanhoth) [inline app/src/main/java/org/gdg/frisbee/android/view/MyView.java]: Forgot one in #200 :smile:

**Comment by stefanhoth** [comment]
Waiting for @friedger @mauimauer to enlighten us.

File: app/src/main/java/org/gdg/frisbee/android/view/MyView.java (hunk 3 above)
Comment (tasomaniac) [inline app/src/main/java/org/gdg/frisbee/android/view/MyView.java]: Appearently. :smile:

**Comment by tasomaniac** [comment]
I also wonder the answer of the question :smile:

File: app/src/main/java/org/gdg/frisbee/android/view/MasonryView.java (hunk 2 above)
Comment (HenriqueRocha) [inline app/src/main/java/org/gdg/frisbee/android/view/MasonryView.java]: Man, I laughed so much with this GIF! :D

**Comment by mauimauer** [comment]
Some things better stay shrouded in mystery...

**Comment by stefanhoth** [comment]
@mauimauer I'll take this as "I have no objection on killing those and it shouldn't impact the app". Also I read "I was drunk when I wrote those and can't remember anything." :smirk_cat:

**Comment by tasomaniac** [comment]
haha :smile:

**Pull request merged**